contracts/timealarms/src/dispatcher.rs

```diff
@@ -25,16 +26,17 @@ impl OracleAlarmDispatcher {
             emitter: Emitter::of_type(EVENT_TYPE),
         }
     }
-}
-impl AlarmDispatcher for OracleAlarmDispatcher {
-    fn send_to(mut self, id: Id, addr: Addr) -> Result<Self, AlarmError> {
+
+    pub fn send_to(mut self, id: Id, addr: Addr) -> Result<Self, ContractError> {
```

Comment:
why do we need it `pub`?

Reply:
earlier it was in the impl of a trait, so it was `pub`. It's used outside in the `timealarms::alarms`, don't see the problem.